curator-framework/src/main/java/org/apache/curator/framework/api/ErrorListenerPathAndBytesable.java

```diff
@@ -1,3 +1,21 @@
+/**
```

Comment:
This is not actually part of this PR. I guess the license was missing on this file. Might as well just leave it here.